md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
```

Comment:
Top-level request: http://archive.fortune.com/galleries/2008/fortune/0804/gallery.bestadvice.fortune/7.html. I think the soda company CEO was beaten to this advice by a couple millennia, but IMO it's important.